solutions/project/spoke-project-unclass/network/psc.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeAddress
+metadata:  
+  name: apis-private-ip
+  namespace: networking
+  annotations:
+      cnrm.cloud.google.com/project-id: project-id # kpt-set: ${project-id}
+      config.kubernetes.io/depends-on: compute.cnrm.cloud.google.com/namespaces/networking/ComputeNetwork/project-id-global-vpc1-vpc # kpt-set: compute.cnrm.cloud.google.com/namespaces/networking/ComputeNetwork/${project-id}-global-vpc1-vpc
+      cnrm.cloud.google.com/blueprint: 'kpt-fn'
+spec:  
+  addressType: INTERNAL
+  description: a test global address
+  location: global
+  ipVersion: IPV4
+  purpose: PRIVATE_SERVICE_CONNECT
+  address: 10.255.255.254
+  networkRef:
+    name: project-id # kpt-set: ${project-id}-global-vpc1-vpc
+    namespace: networking
+---
+# Private endpoint forwarding rule
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeForwardingRule
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: project-id # kpt-set: ${project-id}
+    cnrm.cloud.google.com/blueprint: 'kpt-fn'
+  name: endpoint1
+  namespace: networking
+spec:
+  ipAddress:
+    addressRef:
+      external: 10.255.255.254
```

Comment:
any particular reason why using an `external` reference here instead of a `name`

Reply:
Uday copied the code from me! lol ;) I had probably just used the example to quickly build up the environment before refining the code.

Reply:
changed to use name